.changeset/sweet-dancers-exist.md

```diff
@@ -0,0 +1,7 @@
+---
+"@comet/admin": patch
+---
+
+Deprecate the `ToolbarActionButton` component
+
+Use `Button` from `@comet/admin` with the `responsive` prop instead.
```

Comment:
I guess you added the "unnecessary" "from `@comet/admin`" to prevent confusion with MUI, right?

Reply:
Yes 🙂

Reply:
Would you prefer this or should we just remove it? 

```suggestion
Use `Button` (from `@comet/admin`) with the `responsive` prop instead.
```